src/shipit_signoff/shipit_signoff/.swagger-codegen-ignore

```diff
@@ -0,0 +1,23 @@
+# Swagger Codegen Ignore
```

Comment:
It performs a similar function to .gitignore, just for the code generation. We could add it, but it might be useful for multiple people to have the same one